frameworks/Java/activeweb/benchmark_config

```diff
@@ -4,40 +4,23 @@
     "default": {
       "setup_file": "setup",
       "json_url": "/activeweb/json",
-      "plaintext_url": "/activeweb/plaintext",
-      "port": 8080,
-      "approach": "Realistic",
-      "classification": "Platform",
-      "database": "None",
-      "framework": "activeweb",
-      "language": "Java",
-      "orm": "Raw",
-      "platform": "ActiveWeb",
-      "webserver": "Resin",
-      "os": "Linux",
-      "database_os": "Linux",
-      "display_name": "ActiveWeb",
-      "notes": "",
-      "versus": "servlet"
-    },
-    "raw": {
-      "setup_file": "setup",
       "db_url": "/activeweb/db",
       "query_url": "/activeweb/queries?queries=",
       "fortune_url": "/activeweb/fortunes",
       "update_url": "/activeweb/updates?queries=",
+      "plaintext_url": "/activeweb/plaintext",
       "port": 8080,
       "approach": "Realistic",
-      "classification": "Platform",
+      "classification": "Full-stack",
```

Comment:
ping @msmith-techempower - can you take a look at all the changes to these files? I think you're better at understanding the intention behind the keys than I am, and multiple keys are being changed. Of note, I've never seen `platform="servlet"` that I can recall, I am unsure if jdbc (which uses activerecord concept I think) is a micro ORM or a full ORM, I still stink at distinguishing what the intention is behind platform vs full-stack, etc. Basically, I'm totally lost as to if this file is correct ;-)